smartsim/_core/entrypoints/colocated.py

```diff
@@ -54,7 +54,9 @@
 SIGNALS = [signal.SIGINT, signal.SIGTERM, signal.SIGQUIT, signal.SIGABRT]
 
 
-def handle_signal(signo: int, frame: t.Optional[FrameType]) -> None:
+def handle_signal(signo: int, _frame: t.Optional[FrameType]) -> None:
+    if not signo:
+        logger.warning("Received signal with no signo")
```

Comment:
I think I ran out of creativity. Without that line, the linter would complain about unused parameters. It seems impossible to have the handler be called without a signal number, but it was the easiest way to avoid another suppression and does provide "real information"

I'm open to suggestions (e.g. pass to cleanup and actually inspect signal to understand whats happening)